config.inc.php

```diff
@@ -86,6 +86,8 @@ function language_hook($PALANG, $language) {
 $CONF['database_user'] = 'postfix';
 $CONF['database_password'] = 'postfixadmin';
 $CONF['database_name'] = 'postfix';
+// Default mysql port for mysqli connection
+$CONF['database_port'] = '3308';
```

Comment:
PostgreSQL also looks at $CONF[database_port], and from looking at the code in functions.inc.php, I'm afraid this change would cause breakage for all PostgreSQL users because PostfixAdmin would always try to connect to the MySQL port ;-)

I'm a bit undecided what the best way to handle this is - possible options include
* adding a $CONF[database_port_mysql] (argh, another config option)
* having $CONF[database_port] = '0' as default, which would mean to use the default port of each database

BTW: For mysql, something like $CONF[database_host] = '127.0.0.1:3308' worked. Does this also work for mysqli?

Reply:
* leave the port undefined by default.
 * pass in the default value to the appropriate DB driver if it's not defined in the config. ?